tests/config_dhcp_relay_test.py

```diff
@@ -0,0 +1,259 @@
+from unittest import mock
+from click.testing import CliRunner
+
+config_dhcp_relay_add_output = """\
+Added DHCP relay address [{}] to Vlan1000
+Restarting DHCP relay service...
+"""
+config_dhcp_relay_del_output = """\
+Removed DHCP relay address [{}] from Vlan1000
+Restarting DHCP relay service...
+"""
+expected_dhcp_relay_add_config_db_output = {
+    "ipv4": {
+        "dhcp_servers": [
+            "192.0.0.1", "192.0.0.2", "192.0.0.3", "192.0.0.4", "192.0.0.5"
+        ],
+        "dhcpv6_servers": [
+            "fc02:2000::1", "fc02:2000::2"
+        ],
+        "vlanid": "1000"
+    },
+    "ipv6": {
+        "dhcpv6_servers": [
+            "fc02:2000::1", "fc02:2000::2", "fc02:2000::3"
+        ]
+    }
+}
+expected_dhcp_relay_del_config_db_output = {
+    "ipv4": {
+        "dhcp_servers": [
+            "192.0.0.1", "192.0.0.2", "192.0.0.3", "192.0.0.4"
+        ],
+        "dhcpv6_servers": [
+            "fc02:2000::1", "fc02:2000::2"
+        ],
+        "vlanid": "1000"
+    },
+    "ipv6": {
+        "dhcpv6_servers": [
+            "fc02:2000::1", "fc02:2000::2"
+        ]
+    }
+}
+expected_dhcp_relay_add_multi_config_db_output = {
+    "ipv4": {
+        "dhcp_servers": [
+            "192.0.0.1", "192.0.0.2", "192.0.0.3", "192.0.0.4", "192.0.0.5", "192.0.0.6", "192.0.0.7"
+        ],
+        "dhcpv6_servers": [
+            "fc02:2000::1", "fc02:2000::2"
+        ],
+        "vlanid": "1000"
+    },
+    "ipv6": {
+        "dhcpv6_servers": [
+            "fc02:2000::1", "fc02:2000::2", "fc02:2000::3", "fc02:2000::4", "fc02:2000::5"
+        ]
+    }
+}
+
+IP_VER_TEST_PARAM_MAP = {
+    "ipv4": {
+        "command": "helper",
+        "ips": [
+            "192.0.0.5",
+            "192.0.0.6",
+            "192.0.0.7"
+        ],
+        "exist_ip": "192.0.0.1",
+        "nonexist_ip": "192.0.0.8",
+        "invalid_ip": "192.0.0",
+        "table": "VLAN"
+    },
+    "ipv6": {
+        "command": "destination",
+        "ips": [
+            "fc02:2000::3",
+            "fc02:2000::4",
+            "fc02:2000::5"
+        ],
+        "exist_ip": "fc02:2000::1",
+        "nonexist_ip": "fc02:2000::6",
+        "invalid_ip": "fc02:2000:",
+        "table": "DHCP_RELAY"
+    }
+}
+
+
+@pytest.fixture(scope="module", params=["ipv4", "ipv6"])
+def ip_version(request):
+    """
+    Parametrize Ip version
+    Args:
+        request: pytest request object
+    Returns:
+        Ip version needed for test case
+    """
+    return request.param
+
+
+@pytest.fixture(scope="module", params=["add", "del"])
+def op(request):
+    """
+    Parametrize operate tpye
+    Args:
+        request: pytest request object
+    Returns:
+        Operate tpye
+    """
+    return request.param
+
+
+class TestConfigDhcpRelay(object):
+    @classmethod
+    def setup_class(cls):
+        os.environ['UTILITIES_UNIT_TESTING'] = "1"
+        print("SETUP")
+
+    def test_config_dhcp_relay_add_del_with_nonexist_vlanid(self, ip_version, op):
```

Comment:
Do we have code to handle situation like `remove_vlan_while_dhcp_config_exist`?

Reply:
Fixed

Reply:
Fixed by https://github.com/sonic-net/sonic-utilities/pull/2660